src/structures/guild/GuildMember.ts

```diff
@@ -0,0 +1,43 @@
+import { APIGuildMember, Snowflake } from 'discord-api-types';
+import { Guild } from '../guild/Guild';
+import { User } from '../User';
+
+export class GuildMember {
+	public user?: User;
```

Comment:
are you sure that this is nullable ?

Reply:
Yes : https://discord.com/developers/docs/resources/guild#guild-member-object-guild-member-structure